third_party/xla/xla/stream_executor/gpu/gpu_cudamallocasync_allocator.cc

```diff
@@ -246,10 +255,11 @@ GpuCudaMallocAsyncAllocator::GpuCudaMallocAsyncAllocator(
     // Set the previous pools access to the current GPU.
     map.location.id = platform_device_id.value();
 
-    VLOG(2) << "Set access to the pool id: " << i
+    VLOG(2) << "Set access to the pool id: " << (*all_ids_)[i].value()
```

Comment:
use a temp var to save `(*all_ids_)[i].value()`, and use it below?

Reply:
create a var `int previous_pool_id = (*all_ids_)[i].value();`.